evalml/pipelines/components/estimators/regressors/svm_regressor.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from sklearn.svm import SVR
+from skopt.space import Real
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+
+
+class SVMRegressor(Estimator):
+    """Support Vector Machine Regressor."""
+    name = "SVM Regressor"
+    hyperparameter_ranges = {
+        "C": Real(0, 10),
+        "kernel": ["linear", "poly", "rbf", "sigmoid", "precomputed"],
+        "gamma": ["scale", "auto"]
```

Comment:
I know we've been having conversations around imbalanced data sets so if you want to add the option of `balanced` to `class_weight` in the hyperparameters, that could be interesting, but not required

Reply:
Yeah, I think this would be really cool to do, but prolly after we figure out how to add weighted args to our components/pipelines in general